UserCard.jsx

```diff
@@ -0,0 +1,33 @@
+function UserCard(props) {
+  const [avatar, setAvatar] = React.useState('Loading...')
+  const [firstName, setFirstName] = React.useState('Loading...')
+  const [lastName, setLastName] = React.useState('Loading...')
+  const [email, setEmail] = React.useState('Loading...')
+
+  React.useEffect(() => {
+    async function getUsers(userId) {
+      const response = await fetch(`https://reqres.in/api/users/${userId}`)
+      const data = await response.json()
+
+
+
+      setAvatar(data.data.avatar)
+      setFirstName(data.data.first_name)
+      setLastName(data.data.last_name)
+      setEmail(data.data.email)
+
+    }
+    getUsers(props.userId)
+  }, [props.userId])
```

Comment:
Nice work remembering to include this dependency in the dependency array - this is what makes the `re-render` possible for your search functionality!